src/components/QuickOpenModal.js

```diff
@@ -302,6 +340,10 @@ export class QuickOpenModal extends Component<Props, State> {
       return null;
     }
 
+    if (results) {
```

Comment:
Based on my note above you should do this also even if you were to do it differently I don't think you should be mutating `results`:

```js
let highlightedResults;
if (results) {
  highlightedResults = this.fuzzyhighlight(query, results);
}
```